Source/SVGKImage.h

```diff
@@ -290,6 +290,9 @@ typedef void (^SVGKImageAsynchronousLoadingDelegate)(SVGKImage* loadedImage, SVG
 /*! returns all the individual CALayer's in the full layer tree, indexed by the SVG identifier of the SVG node that created that layer */
 - (NSDictionary*) dictionaryOfLayers;
 
+/*! returns all the individual CALayer's in the full layer tree as an array */
+- (NSArray*) arrayOfLayers;
```

Comment:
Why is this needed?

Doesn't dictionaryOfLayers provide 100% of this already?

I'm OK with adding features, but I don't want to add anything to SVGKImage.h unless it's essential -- I really want us to move features OUT of SVGKIimage.h and into their own classes!